tests/other/test_information_source.py

```
@@ -0,0 +1,33 @@
+        )
+        
+    def test_load_information_source_command(self):
+        """Test if the load_information_source command runs without errors."""
+        try:
+            call_command("load_information_source")
+        except Exception as e:
+            self.fail(f"load_information_source command failed: {e}")
+
+    def test_abbreviation_unique_constraint(self):
+        """Test if the abbreviation field is unique."""
+        with self.assertRaises(Exception):
```

Ruff (0.8.2)

`assertRaises(Exception)` should be considered evil

(B017)